packages/sdk/typescript/subgraph/src/mapping/utils/dayUpdates.ts

```diff
@@ -27,6 +27,8 @@ export function getEventDayData(event: ethereum.Event): EventDayData {
     eventDayData.dailyPayoutAmount = ZERO_BI;
     eventDayData.dailyHMTTransferCount = ZERO_BI;
     eventDayData.dailyHMTTransferAmount = ZERO_BI;
+    eventDayData.uniqueSenders = ZERO_BI;
```

Comment:
can you please rename to follow the same structure dailyUniqueSenders and dailyUniqueReceivers?